iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/voter/AffOrgMatchVotersFactory.java

```diff
@@ -15,43 +15,62 @@ private AffOrgMatchVotersFactory() { }
     
     //------------------------ LOGIC --------------------------
     
-    public static AffOrgMatchVoter createNameCountryStrictMatchVoter() {
+    public static AffOrgMatchVoter createNameCountryStrictMatchVoter(float matchStrength) {
         
-        return new CompositeMatchVoter(ImmutableList.of(
+        CompositeMatchVoter voter = new CompositeMatchVoter(ImmutableList.of(
                 new CountryCodeStrictMatchVoter(), 
                 new NameStrictWithCharFilteringMatchVoter(ImmutableList.of(',', ';'))));
         
+        voter.setMatchStrength(matchStrength);
```

Comment:
Well, probably not, but there are other Voters that don't have it always set. And passing it sometimes in constructors and sometimes via setters would make a mess.